hybrid/clusterlink/sasl_ssl_source_cluster/README.md

```diff
@@ -101,7 +101,7 @@ kubectl -n destination create secret generic rest-credential \
     --from-file=basic.txt=$TUTORIAL_HOME/rest-credential.txt
     
 kubectl -n destination create secret generic password-encoder-secret \
-    --from-file=password_encoder_secret=$TUTORIAL_HOME/password-encoder-secret.txt
+    --from-file=password-encoder.txt=$TUTORIAL_HOME/password-encoder-secret.txt
```

Comment:
Same here

Reply:
updated the format in password-encoder-secret.txt file